test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1

```diff
@@ -339,4 +339,191 @@ Left Center Right
 			$output = $ps.Invoke()
 			$output.Replace("`n","").Replace("`r","") | Should -BeExactly $expected
 		}
+
+		It "Format-Table should correctly render headers that span multiple rows: <variation>" -TestCases @(
+			@{
+				variation = "4 row, 1 row, 2 row";
+				ps1xml = @"
+<Configuration>
+	<ViewDefinitions>
+		<View>
+			<Name>Test.Format</Name>
+			<ViewSelectedBy>
+				<TypeName>Test.Format</TypeName>
+			</ViewSelectedBy>
+			<TableControl>
+				<TableHeaders>
+					<TableColumnHeader>
+						<Label>LongLongHeader</Label>
+						<Width>4</Width>
+					</TableColumnHeader>
+					<TableColumnHeader>
+						<Label>Header2</Label>
+						<Width>7</Width>
+					</TableColumnHeader>
+					<TableColumnHeader>
+						<Label>Header3</Label>
+						<Width>4</Width>
+					</TableColumnHeader>
+				</TableHeaders>
+				<TableRowEntries>
+					<TableRowEntry>
+						<TableColumnItems>
+							<TableColumnItem>
+								<PropertyName>First</PropertyName>
+							</TableColumnItem>
+							<TableColumnItem>
+								<PropertyName>Second</PropertyName>
+							</TableColumnItem>
+							<TableColumnItem>
+								<PropertyName>Third</PropertyName>
+							</TableColumnItem>
+						</TableColumnItems>
+					</TableRowEntry>
+				</TableRowEntries>
+			</TableControl>
+		</View>
+	</ViewDefinitions>
+</Configuration>
+"@; expectedTable = @"
+
+Long Header2 Head
+Long         er3
+Head
+er
+---- ------- ----
```

Comment:
I believe we should have a test for edge case - one column.

Reply:
Will add

Reply:
I'm also going to turn the ps1xml into a template since most of it is the same